README.rst

```diff
@@ -100,6 +101,27 @@ Running unit tests::
 Tox will take care of building a faux installation environment, and ensure that
 the mapped import paths work correctly.
 
+Running from source on old Linux
+================================
+
+If your Linux distribution has a different version of python 3 (such as python
+3.5 in Debian 9), it is recommended to do a user-space install with
+`pyenv <https://github.com/pyenv/pyenv-installer>`_. This allows Electron
+Cash to run completely independently of your system configuration.
+
+1. Install `pyenv <https://github.com/pyenv/pyenv-installer>`_ in your user
+   account. Follow the printed instructions about updating your environment
+   variables and ``.bashrc``, and restart your shell to ensure that they are
+   loaded.
+2. Run ``pyenv install 3.6.9``. This will download and compile that version of
+   python, storing it under ``.pyenv`` in your home directory.
+3. ``cd`` into the Electron Cash directory. Run ``pyenv local 3.6.9`` which inserts
+   a file ``.python-version`` into the current directory.
+4. While still in this directory, run ``pip install pyqt5``.
+5. If you are installing from the source file (.tar.gz or .zip) then you are
```

Comment:
I think this should be `pip3 install pyqt5` .

Reply:
Hmm, it shouldn't matter, since pip will be pip3 when pyenv is correctly active in env variables.

Reply:
Yeah but shouldn't we be teaching good habits here? *shrugs*.